src/Compilers/CSharp/Portable/Parser/Lexer.cs

```diff
@@ -2293,12 +2293,145 @@ private void LexSyntaxTrivia(bool afterFirstToken, bool isTrailing, ref SyntaxLi
                         {
                             return;
                         }
+
+                    case '>':
```

Comment:
Yes.  We would be breaking that scenario.  Up to you guys if you think such a break could realistically happen.   Note: the issue occurs for TS, and we never got a single report about that.